.circleci/config.yml

```diff
@@ -731,17 +675,6 @@ jobs:
           no_output_timeout: 20m
       - store-artifacts
 
-  # Deprecated - use workflows in conjunction with smoke-tests job instead!
-  # This job is manually triggered for now. see .circleci/README.md
-  production-smoke-tests:
```

Comment:
Do we need this?

Reply:
That sounds reasonable, I see that the readme instructions for triggering manually were deleted.

Reply:
They get triggered remotely by SRE. Cannot remove.

Reply:
I am fairly certain what is happening is SRE is calling the [production_smoke_tests](https://github.com/mozilla/fxa/blob/main/.circleci/config.yml#L1052) workflow, which uses the [smoke-tests](https://github.com/mozilla/fxa/blob/main/.circleci/config.yml#L745) job, not the [production-smoke-tests](https://github.com/mozilla/fxa/blob/main/.circleci/config.yml#L736) job here.